docs/src/howToGetStarted.md

```diff
@@ -38,105 +281,267 @@ for f in dataFiles
+
+# remove the temporary data from workers
+undistribute(mapping)
+```
 
+If this was used for the first hypercube dataset, you will see many obvious
+hypercube clusters (although not all, because `k=10` is not enough to capture
+all of the actual 16 clusters):
 
 ```julia
-som2 = initGigaSOM(dfSom, 10, 10)
+clusterMeans
 ```
 
-Train the SOM grid with the initialized SOM object and define the number of training
-rounds (also referred to as *epochs*).
+```
+10×4 Array{Float64,2}:
+  4.84504    -0.0178982   9.96031     10.0188
+  5.12037    10.0034      9.97428     10.0159
+  5.03354    10.0223     10.016       -0.0217869
+  5.10709    -0.0216336   0.00545618  10.0337
+  4.97722    10.0138     -0.0409751   -0.0209958
+ 10.0156      0.0172955   9.968        0.00141821
+ 10.0359      9.93229     0.0751434    9.9335
+  0.0833944   0.0380366   9.93105      0.00187243
+  5.11778    -0.0418081   0.0404327   -0.00459735
+  0.0130978   9.99403    -0.0446802    9.99991
+```
 
+Notably, several of the most used stats functions are available, with a nice frontend.
 
+Distributed median computation (approximative):
 ```julia
- som2 = trainGigaSOM(som2, dfSom, epochs = 10)
+dmedian(di, [1,2,3,4])
+```
+
+The medians are slightly off-center because there is a lot of empty space between the clusters:
+```
+3-element Array{Float64,1}:
+ 6.947097488861494
+ 7.934405685940568
+ 7.069149844215707
+ 2.558892109203585
 ```
 
-Finally, calculate the winner neurons from the trained SOM object:
+`dstat` function also has the bucketed variant
 
+```julia
+dstat_buckets(di, 10, mapping, [1,2,3,4])[2]   # (taking out only the sdevs, for niceness)
+```
+
+We can count there are 4 "nice" clusters, and 6 clusters that span 2 of the
+original clusters, totally giving 16. (validation succeeded!)
+```
+10×4 Array{Float64,2}:
+ 5.09089   0.997824  1.01815   0.980758
+ 5.13971   1.02019   0.977637  1.00124
+ 5.13209   0.974332  1.00058   0.99874
+ 5.11529   0.998166  1.01825   1.01885
+ 5.10542   1.01686   0.975993  0.991992
+ 0.991075  0.993312  1.00667   1.05048
+ 0.996443  1.02699   0.938742  0.98831
+ 0.946917  0.989543  1.0056    0.999609
+ 5.09963   1.00131   0.978803  0.984435
+ 1.00892   0.998226  1.05538   0.994829
+```
 
+The same for the approximate median:
 ```julia
-winners = mapToGigaSOM(som2, dfSom)
+dmedian_buckets(di, 10, mapping, [1,2,3,4])
+```
+
+Medians should be roughly similar to means, but some of our clusters have a
+hole that the median dodges:
+```
+10×4 Array{Float64,2}:
+  1.97831    -0.0120118    9.98967    10.0161
+  7.99438    10.0263       9.9988     10.0033
+  3.27907     9.98728     10.0254      0.00444198
+  7.91739    -0.0623953   -0.0240277  10.0374
+  2.445      10.0101      -0.0471141  -0.0253346
+ 10.0121      0.00935064   9.94992     0.0459787
+ 10.0512      9.93359      0.0923141   9.91175
+  0.0675462  -0.0142712    9.93406     0.0343599
+  8.09972    -0.0217352    0.0575258  -0.010485
+ -0.0183372  10.0392      -0.115253   10.0101
```

Comment:
Somewhat this tutorial is super long, maybe some conclusions or remarks would help.